tests/FormaterTests.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import os
+from app.Notebook import Cell, Notebook
+from app.Formater import ToNotebook, ToPy
+
+
+class FormaterTests(unittest.TestCase):
+    def test_topy_class_creation(self):
+        topy = ToPy()
+        self.assertIsInstance(topy, ToPy)
+
+    def test_to_notebook_class_creation(self):
+        tonb = ToNotebook()
+        self.assertIsInstance(tonb, ToNotebook)
+
+    def test_topy_empty_constructor_creation(self):
+        topy = ToPy()
+        self.assertEquals((topy.dry_run, topy.overwrite), (False, False))
+
+    def test_to_notebook_empty_constructor_creation(self):
+        tonb = ToNotebook()
+        self.assertEquals((tonb.dry_run, tonb.overwrite), (False, False))
+
+    def test_nb_constructor(self):
+        tonb = ToNotebook(True, True)
+        self.assertEquals((tonb.dry_run, tonb.overwrite), (True, True))
+
+    def test_py_constructor(self):
+        topy = ToPy(True, True)
+        self.assertEquals((topy.dry_run, topy.overwrite), (True, True))
+
+    def test_topy_construct_path_method(self):
+        topy = ToPy()
+        path = topy.construct_output_path('a.exe')
+        self.assertEquals(path, 'a.py')
+
```

Comment:
The expected result should be encoded in the testname.

Eg test_construct_output_path_changes_extension_to_dotpy